lockdown/core.py

```diff
@@ -158,77 +215,138 @@ async def lockdownset(self, ctx):
             get_confirmation = await self.config.guild(guild).confirmation_message()
             chan = ""
             for channel in get_channel:
-                chan += "<#{}> - {}\n".format(channel, channel)
+                chan += "'{}' - <#{}>\n".format(channel, channel)
+            # for r in get_role:
+            #     role += "`{}` - <@&{}>\n".format(r, r)
 
-            embed = discord.Embed(
-                color=await ctx.embed_color(), title="Current channels:", description=chan
+            e = discord.Embed(
+                color=await ctx.embed_color(), title="Lockdown Config:", description=chan,
             )
 
-            embed.add_field(name="Lock Message:", value=get_lock, inline=False)
+            # e.add_field(name="Channels", value=chan, inline=False)
+            e.add_field(name="Lock Message:", value=get_lock, inline=False)
+            e.add_field(name="Unlock Message:", value=get_unlock, inline=False)
+            e.add_field(name="Unlock Embed", value=check_embed, inline=False)
+            e.add_field(name="Confirmation:", value="enabled" if get_confirmation else "disabled")
+            # e.add_field(name="Non-Default Roles", value=role, inline=False)
 
-            embed.add_field(name="Unlock Message:", value=get_unlock, inline=False)
+            await ctx.send(embed=e)
 
-            embed.add_field(
-                name="Confirmation:", value="enabled" if get_confirmation else "disabled"
+    @lockdownset.command("embed")
+    async def send_embed(self, ctx: commands.Context, *, default: bool = None):
+        """
+        Indicate if you would like to send an embed or not on unlock
+        """
+        guild = ctx.guild
+        if default is None:
+            await ctx.send(
+                "Pass True or False. True sends an embed on unlock, False silences the embed and sends plain text on unlock. It's false by default"
             )
-
-            await ctx.send(embed=embed)
+            return
+        if default is False:
+            await ctx.send(
+                "Alright, will send unlock message in plain text if it's set (`[p]lockdownset unlockmsg`)"
+            )
+            await self.config.guild(guild).embed_set.set(default)
+            return
+        await self.config.guild(guild).embed_set.set(default)
+        await ctx.send(
+            "Will send an embed with your unlock message as the body, if you've set it up"
+        )
 
     @lockdownset.command()
-    async def channel(self, ctx, channel: discord.TextChannel):
+    async def addchan(self, ctx: commands.Context, *channel: discord.TextChannel):
         """
-        Toggles lockdown status for a particular channel
+        Adds channel to list of channels to lock/unlock
+        You can add as many as needed
+        Example: `;;lds addchan general support bot-commands`
+        IDs are also accepted.
         """
-        status = "Error"
-        if channel.id not in await self.config.guild(ctx.guild).channels():
```

Comment:
I should've kept this line actually to prevent double additions
*noting for the PR*